packages/contracts/src/plugins/governance/majority-voting/addresslist/build-metadata.json

```diff
@@ -1,12 +1,65 @@
 {
   "ui": "",
-  "change": "",
+  "change": "- The ability to create a proposal now depends on the membership status of the current instead of the snapshot block.",
   "pluginSetupABI": {
-    "prepareInstallation": [
-      "tuple(uint8 votingMode, uint64 supportThreshold, uint64 minParticipation, uint64 minDuration, uint256 minProposerVotingPower) votingSettings",
-      "address[] members"
-    ],
-    "prepareUpdate": [""],
-    "prepareUninstallation": [""]
+    "prepareInstallation": {
+      "description": "The information required for the installation of build 2.",
+      "inputs": [
+        {
+          "components": [
+            {
+              "internalType": "enum MajorityVotingBase.VotingMode",
+              "name": "votingMode",
+              "type": "uint8",
+              "description": "A parameter to select the vote mode. In standard mode (0), early execution and vote replacement are disabled. In early execution mode (1), a proposal can be executed early before the end date if the vote outcome cannot mathematically change by more voters voting. In vote replacement mode (2), voters can change their vote multiple times and only the latest vote option is tallied."
+            },
+            {
+              "internalType": "uint32",
+              "name": "supportThreshold",
+              "type": "uint32",
+              "description": "The support threshold value. Its value has to be in the interval [0, 10^6] defined by `RATIO_BASE = 10**6`."
```

Comment:
I would make it less scientific and more human. If we're losing control of the install flow because this is generic, people should know that they need to enter a percentage that goes from `0 to 1000000`

Reply:
Most likely, the upcoming modular UI discussion will require the definition of some kind of "UI types". i.e. `percentage`, `timestamp`, `time`, etc to help the UI

Reply:
See here: https://github.com/aragon/osx/pull/375/files#r1200096582